lib/assessment.md

```diff
@@ -41,7 +55,21 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+```.big {
+
+}
+
+#contentPane {
+
+}
+
+.important.red {
 
+}
+
+p:div {
```

Comment:
I don't think this last one is valid CSS. 

One possible answer would be to write `div > p`